src/Language/PureScript/Bundle.hs

```diff
@@ -638,17 +644,20 @@ codeGen optionsMainModule optionsNamespace ms outFileOpt = (fmap sourceMapping o
   wrap :: String -> [JSStatement] -> [JSStatement]
   wrap mn ds =
     [
-    JSMethodCall (JSExpressionParen lf (JSFunctionExpression JSNoAnnot JSIdentNone JSNoAnnot
-                                                (JSLOne (JSIdentName JSNoAnnot "exports")) JSNoAnnot
-                                                (JSBlock sp (lfHead ds) lf)) -- \n not quite in right place
+    JSMethodCall (JSExpressionParen lf (JSFunctionExpression JSNoAnnot JSIdentNone JSNoAnnot (JSLOne (JSIdentName JSNoAnnot "$PS")) JSNoAnnot
+                                                (JSBlock sp (addModuleExports ds) lf)) -- \n not quite in right place
                                     JSNoAnnot)
                   JSNoAnnot
-                  (JSLOne (JSAssignExpression (moduleReference JSNoAnnot mn) (JSAssign sp)
-                            (JSExpressionBinary (moduleReference sp mn) (JSBinOpOr sp) (emptyObj sp))))
+                  (JSLOne (JSIdentifier JSNoAnnot optionsNamespace))
                   JSNoAnnot
                   (JSSemi JSNoAnnot)
     ]
     where
+      addModuleExports :: [JSStatement] -> [JSStatement]
+      addModuleExports [] = lfHead moduleExports 
+      addModuleExports (x:xs) = lfHead [x] ++ moduleExports ++ xs
+      moduleExports = [JSExpressionStatement (JSAssignExpression (innerModuleReference lfsp mn) (JSAssign sp) (JSExpressionBinary (innerModuleReference sp mn) (JSBinOpOr sp) (emptyObj sp))) (JSSemi JSAnnotSpace), JSVariable lfsp (JSLOne $ JSVarInitExpression (JSIdentifier sp "exports") $ JSVarInit sp (innerModuleReference sp mn)) $ JSSemi JSNoAnnot ]
```

Comment:
Last nitpick and then I think this is good to merge: this line is too long. Can you please put each element of this list on its own line and use indentation similar to how it has been used elsewhere in this file (such as in the main body of `wrap` just above here)?